#!/usr/bin/env python
PACKAGE = "aerial_robot_perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lowest_margin",             int_t,    0, "lowest margin",             10,     0, 100)
gen.add("object_height",             double_t, 0, "object height",             0.2,  0.0, 1.0)
gen.add("target_object_area",        double_t, 0, "target object area",        0.06, 0.0, 1.0)
gen.add("target_object_area_margin", double_t, 0, "target object area margin", 0.02, 0.0, 1.0)

exit(gen.generate(PACKAGE, "aerial_robot_perception", "RectangleDetection"))